templates/index.test.js

```diff
@@ -0,0 +1,25 @@
+function content(folderName) {
+  return (
+
+`import React, { Component } from 'react';
```

Comment:
These ticks are new Node; could this be a problem? What minimum version of Node does this constrain us to?

Reply:
@jalada - constrains us to NodeJS v4.0.0 (2015-09-08) https://kangax.github.io/compat-table/es6/
Is that ok or should I change?

Reply:
Showing my age! That's fine.